azure-pipelines/end-to-end-tests-dir/fetch.ps1

```diff
@@ -1,101 +0,0 @@
-. $PSScriptRoot/../end-to-end-tests-prelude.ps1
```

Comment:
I don't think outright deleting this test is OK. Yes, it means the test will have to be fixed when we change the table in the tool, but this is testing important aspects of `fetch` which aren't parsing that XML file.

Reply:
I changed the existing test instead